tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -36,13 +36,7 @@ To connect to your TiDB cluster via standard connection, perform the following s
 
     3. Click **Create Filter** to confirm the changes.
 
-4. Use a SQL client to connect to TiDB.
-
-    {{< copyable "shell" >}}
-
-    ```shell
-    mysql -u root -h <endpoint> -P <port number> -p
-    ```
+4. Under **Step 2: Connect with a SQL client** in the dialog box, click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```

Comment:
```suggestion
4. In the **Connect to TiDB** dialog box displayed in "Step 2: Click **Connect**", click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```

Reply:
"Step 2: Click **Connect**" is in the the **Connect to TiDB** dialog box.